src/EditorFeatures/Test/Formatting/InferredIndentationTests.cs

```diff
@@ -15,46 +16,57 @@ namespace Microsoft.CodeAnalysis.Editor.UnitTests.Formatting
     [UseExportProvider]
     public class InferredIndentationTests
     {
-        [Fact]
-        public async Task BlankFileMatchesWorkspaceSettings()
+        public static IEnumerable<object[]> LineEndings => new[] { new[] { "\r\n" }, new[] { "\n" } };
+
+        [Theory]
+        [MemberData(nameof(LineEndings))]
+        public async Task BlankFileMatchesWorkspaceSettings(string lineEnding)
         {
             using var testWorkspace = CreateWithLines(
+                lineEnding,
                 "");
             var options = await testWorkspace.CurrentSolution.Projects.Single().Documents.Single().GetOptionsAsync();
 
+            Assert.Equal(FormattingOptions.NewLine.DefaultValue, options.GetOption(FormattingOptions.NewLine));
             Assert.Equal(FormattingOptions.UseTabs.DefaultValue, options.GetOption(FormattingOptions.UseTabs));
         }
 
-        [Fact]
-        public async Task SingleLineWithTab()
+        [Theory]
+        [MemberData(nameof(LineEndings))]
+        public async Task SingleLineWithTab(string lineEnding)
         {
             using var testWorkspace = CreateWithLines(
+                lineEnding,
                 "class C",
                 "{",
                 "\tvoid M() { }",
                 "}");
             var options = await testWorkspace.CurrentSolution.Projects.Single().Documents.Single().GetOptionsAsync();
 
+            Assert.Equal(lineEnding, options.GetOption(FormattingOptions.NewLine));
             Assert.True(options.GetOption(FormattingOptions.UseTabs));
         }
 
-        [Fact]
-        public async Task SingleLineWithFourSpaces()
+        [Theory]
+        [MemberData(nameof(LineEndings))]
+        public async Task SingleLineWithFourSpaces(string lineEnding)
         {
             using var testWorkspace = CreateWithLines(
+                lineEnding,
                 "class C",
                 "{",
                 "    void M() { }",
                 "}");
             var options = await testWorkspace.CurrentSolution.Projects.Single().Documents.Single().GetOptionsAsync();
 
+            Assert.Equal(lineEnding, options.GetOption(FormattingOptions.NewLine));
```

Comment:
📝 These assertions are failing because the option is not part of adaptive formatting.